packages/google_maps_flutter/google_maps_flutter_web/lib/src/convert.dart

```diff
@@ -351,14 +282,14 @@ gmaps.PolygonOptions _polygonOptionsFromPolygon(
     gmaps.GMap googleMap, Polygon polygon) {
   List<gmaps.LatLng> path = [];
   polygon.points.forEach((point) {
-    path.add(_latLngToGmLatLng(point));
+    path.add(_latLngToGmLatLng(point)!);
```

Comment:
It's not clear why this and the below are safe (argument for changing the API to return non-null)